runner/steps.ts

```diff
@@ -161,7 +161,9 @@ function runBrowsers(context: Context) {
       // TODO(nevir): Better rationalize run-end and hook.
       await context.emitHook('cleanup');
 
-      return error;
+      if (error) {
+        throw error;
```

Comment:
I've always strongly believed in only using Errors as exceptions as a best practice. They're much more powerful representations, and I believe most people assume that thrown/propagated errors are Error objects and not strings so you end up causing less surprises and cascading bugs in error handling.

If `runTests()` is ever exposed or called in a different way in the future, then I could definitely see this causing problems.